spec/support/twilio_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module TwilioHelpers
+  def twilio
+    @_twilio ||= Twilio::REST::Client.new TWILIO_SID, TWILIO_TOKEN
+  end
+
+  def get_otp(after_date=nil)
+    count = 0
+    max_count = 5
+    twilio.messages.list(date_sent: after_date).first.body[0..5]
```

Comment:
It looks like Twilio actually has 2 messages, a message with the OTP and an automated reply. We may wanna go back to the original method with a regex that extracts the OTP from a matching message so we don't pick up the automated reply.

Reply:
Also, I think the argument we want is `date_sent_after`. This actually does not work for me unless I change this to:

```
date_sent_after: after_date - 30
```

Reply:
Ahh I forgot about that. For new numbers we'll want to turn off the default auto response. That requires creating some custom TwilML code.

Reply:
I created a blank response so Twilio doesn't automatically process and reply to messages. Can you try this again to see if you still have the same problem?

Reply:
`date_sent` should work for all but a few seconds around midnight UTC of each day :). The API call returns all messages from that day in UTC time (it doesn't matter what minute or second is used in the `after_date`). `date_sent_after` is the better choice since there is zero chance for that edge case to occur. Good find!